tools/markdown-magic/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fluid-tools/markdown-magic",
-	"version": "0.1.0",
+	"version": "2.1.0",
 	"private": true,
 	"description": "Contains shared utilities for Markdown content generation and embedding using markdown-magic.",
 	"homepage": "https://fluidframework.com",
```

Comment:
directory (line 10) should be updated to new location

Reply:
This one is staying where it is for now.